### PR TITLE
Allow '/people' in robots.txt

### LINKED ISSUE #3115: Allow '/people' in robots.txt
We want to index /users and /people (lists + reading logs, as they become public)

### Related files

https://github.com/internetarchive/openlibrary/blob/75452cf6f0e3c23e1030395c93e232686ebaa417/static/robots.txt#L3-L4

### PR DESCRIPTION
**Pull request by imskr**
Closes #3115 

### Stakeholders

@mekarpeles

### CODE REVIEW

**Comment by imskr** [comment]
Okay @tfmorris thanks. I forgot that.

**Comment by imskr** [comment]
@mekarpeles Is this good?

**Comment by mekarpeles** [comment]
(1) do we want to specifically `allow`? or Just omit from the robots.txt completely? Any thoughts from the community? Also, sanity check, is there a reason we may not want these pages indexed?

**Comment by mekarpeles** [comment]
P.S. Thank you for this contribution @imskr and @tfmorris for your guidence!

File: static/robots.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 
 User-agent: *
-Disallow: /user
-Disallow: /people
+Allow: /user
+Allow: /people
```
Comment (mekarpeles) [inline static/robots.txt]:
```suggestion
```

**Comment by hornc** [comment]
`/user` seems to be a redirect to `/people`

I believe `/people` was excluded from robots.txt deliberately, I think this is a pretty big change to have simply worded only as a technical change. 

I imagine this will add a whole lot of blank user pages to the index
e.g. https://openlibrary.org/people/ptrodri

I don't have enough information to evaluate this in terms of requirements, it's just an observation and FYI after wondering what `\user` was used for -- it's a redirect / alias to `/people`. This PR meets all requirements as stated.

**Pull request merged**